README.md

```diff
@@ -1,4 +1,9 @@
-# Inspektor Gadget
+<h1 align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="docs/images/logo/logo-horizontal.png">
+    <img src="docs/images/logo/logo-horizontal-dark.png" alt="Inspektor Gadget" width="80%">
```

Comment:
No, there isn't. I don't have a strong opinion here. Do you suggest using SVG? Why?

Reply:
I see that everyone else is using png as well, so I suppose it's just fine.

Reply:
> Do they parse project README to get the logo for https://ebpf.io/applications?
>
No, they don't.

>  Otherwise, we can perfectly have a SVG in the README and give a PNG for their website?
>
I would prefer to use the same image everywhere.

To conclude, I will go ahead with this PR as it is. We can change to SVG in the future if we find an advantage in doing so.